### ISSUE TITLE
.vtoy not visible

### ISSUE THREAD

**Issue by amervelic** [ventoy/Ventoy#932]
I made a .vdi disk in virtualbox (https://www.ventoy.net/en/plugin_vtoyboot.html)
however .vtoy is not visible when the ventoy starts. it is an ubuntu server 20.04

**Comment by ventoy** [comment]
please check whether you use ventoy.json and specify the search root
Give some photo about your files/directories in your USB.

**Comment by amervelic** [comment]
now visible but the following error has occurred
"System BootOrder not found Could not read \EFI\: Invalid parameter.

**Comment by amervelic** [comment]
success ! I reinstalled the ubuntu server 20.04.2 but now in virtual disk format ".vhd" and it works ok. thanks.

**Comment by candrapersada** [comment]
why zorin os core based ubuntu only shows zorin or (initramfs) on screen when booting using Legacy BIOS?

**Comment by ventoy** [comment]
Only distros listed in the table were tested.
https://www.ventoy.net/en/plugin_vtoyboot.html

**Comment by candrapersada** [comment]
but why if using UEFI can boot into zorin os normally?

**Comment by ventoy** [comment]
Not in the list means that: 
Maybe it can work
Maybe it can not work. 
Maybe it can work only in Legacy BIOS mode
Maybe it can work only in UEFI mode
......

**Comment by candrapersada** [comment]
I also tried it on Ubuntu 20.04.2.0 LTS but the problem is still the same, it can't boot into Legacy BIOS mode and can only boot into UEFI mode, why?

**Comment by ventoy** [comment]
Did it show the following warning when you run vtoyboot.sh in the VM?
```
[WARNING] ##################################################################
[WARNING] #### This vhd/vdi/raw file will only be bootable in UEFI mode ####
[WARNING] ##################################################################
```

**Comment by candrapersada** [comment]
no, I just asked why Linux based ubuntu 21.04 can run using Legacy BIOS like Cinnamon Remix 21.04?

**Comment by eL-Squaz** [comment]
so do we change the actual type "vdi" (drive-file) to .vtoy? or our ISO's?
because when i start it up, it also tells me this same error (original post) when i change the red box  icon vdi file to have vdi.vtoy suffix... doesnt recognize vtoy.. and I did get a full positive install on the vtoyboot package and everything when logged in on the distro. (I'm using the 'manjaro-xfce-21.1.4-210927-linux513.iso' image)
